test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -174,6 +176,7 @@
       app
       [jetty9-service/jetty9-service
        webrouting-service/webrouting-service
+       scheduler-service/scheduler-service
        status-service]
       (merge status-service-config
         {:web-router-service {:puppetlabs.trapperkeeper.services.status.status-service/status-service "/alternate-status"}})
```

Comment:
@jpinsonault you could use this:

https://github.com/puppetlabs/trapperkeeper/blob/master/test/puppetlabs/trapperkeeper/testutils/logging.clj#L210-L220

Then you'd actually be able to de-json the string from the log message.

And I don't think the test would necessarily be racey as long as you slept for some interval longer than the logging interval before looking for the message, and tolerated more than one message in the log output in case the scheduler had fired twice.

That said, I agree it's kind of gross, and could live with the argument that it's not worth testing.  Maybe @rlinehan or @camlow325 has an opinion?